### PR TITLE
MGMT-19571: moving periodic to 3 times a week

### PR DESCRIPTION
**Pull request by eifrach**
Moving Assisted installer periodic testing from 5 times a week to 3 time.

also make it on different days and hours to lower the load / resources during the e2e tests

### CODE REVIEW

**Comment by danmanor** [comment]
@eifrach We need to move all the periodic ZTP jobs as well, the ones that run 5 times a week should be in the master / ocm-2.12 / ocm-2.11 branches across all repos

File: ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.12.yaml [hunk 1]
```diff
@@ -119,7 +119,7 @@ tests:
     cluster_profile: packet-assisted
     workflow: assisted-agent
 - as: subsystem-test-periodic
-  cron: 00 10 * * 0-5
+  cron: 00 20 * * 2,4,6
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.12.yaml:122]: We need also in `master`

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.12.yaml [hunk 2]
```diff
@@ -374,7 +374,7 @@ tests:
       PACKET_PLAN: c3.medium.x86
     workflow: assisted-baremetal-operator-ztp
 - as: e2e-ai-operator-ztp-periodic
-  cron: 00 11 * * 0-5
+  cron: 00 11 * * 1,3,5
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.12.yaml:377]: We need also in `master`

File: ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.12.yaml (hunk 1 above)
Comment (eifrach) [reply ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.12.yaml:122]: done

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.12.yaml (hunk 2 above)
Comment (eifrach) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.12.yaml:377]: done

**Review by danmanor — APPROVED** [review]

**Pull request merged**